test/indexer/data/56a338fe-7554-4b5d-96a2-7df127a7640b.metadata.json

```diff
@@ -194,89 +194,114 @@
     },
     "project.json": {
         "content": {
-            "describedBy": "https://schema.humancellatlas.org/type/project/5.1.0/project",
+            "describedBy": "https://schema.humancellatlas.org/type/project/9.0.0/project",
```

Comment:
I can't find version 9.0.0 but there is version 9.0.2

Reply:
@hannes-ucsc It was in the staging two weeks ago.

Reply:
The link doesn't resolve. Please fix.